app/models/commentaire.rb

```diff
@@ -76,15 +81,12 @@ def file_url
   private
 
   def notify
-    dossier_user_email = dossier.user.email
-    invited_users_emails = dossier.invites.pluck(:email).to_a
-
     # - If the email is the contact email, the commentaire is a copy
     #   of an automated notification email we sent to a user, so do nothing.
     # - If a user or an invited user posted a commentaire, do nothing,
     #   the notification system will properly
     # - Otherwise, a instructeur posted a commentaire, we need to notify the user
-    if !email.in?([CONTACT_EMAIL, dossier_user_email, *invited_users_emails])
+    if !email.in?([CONTACT_EMAIL, *dossier.message_notification_emails])
       notify_user
     end
```

Comment:
Est ce qu'on ne pourrait pas faire plus simple ?

```suggestion
    if sent_by_instructeur?
      notify_user
    end
```

Reply:
J'ai un doute, est ce que les experts ne peuvent pas poser des questions aux usagers ?

Reply:
Erf, t’as raison. Ça n'a pas été proprement géré lors de l'introduction des experts. Je vais corriger (il faut ajouté expert_id aux messages)

Reply:
j'ai corrigé